src/main/java/org/folio/service/manager/export/ExportManagerImpl.java

```diff
@@ -203,19 +203,26 @@ private ExportResult getExportResult(AsyncResult<Object> asyncResult, ExportPayl
     } else {
       LOGGER.info("Export has been successfully passed");
       if (exportPayload.isLast()) {
-        if (exportPayload.getExportedRecordsNumber() == 0) {
-          return ExportResult.failed(ErrorCode.NOTHING_TO_EXPORT);
-        } else if (exportPayload.getFailedRecordsNumber() > 0) {
-          errorLogService.saveGeneralError("Export is finished with errors, some records are failed to export, number of failed records: " + exportPayload.getFailedRecordsNumber(), exportPayload.getJobExecutionId(), exportPayload.getOkapiConnectionParams().getTenantId());
-          return ExportResult.completedWithErrors();
-        } else {
-          return ExportResult.completed();
-        }
+        return getProperExportResultIfBatchIsLast(exportPayload);
       }
       return ExportResult.inProgress();
     }
   }
 
+  private ExportResult getProperExportResultIfBatchIsLast(ExportPayload exportPayload) {
+    if (exportPayload.getExportedRecordsNumber() == 0) {
+      if (vertx.fileSystem().existsBlocking(exportPayload.getFileExportDefinition().getSourcePath())) {
```

Comment:
Use FileStorage service to check the existence of file

Reply:
done